WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/logic/interval/IntervalTransferManager.java

```diff
@@ -71,8 +72,8 @@ public void run() {
 		private void transferIntervals(List<IntervalBase> intervalsToTransfer) {
 			JsonTransferer intervalTransferer = new JsonTransferer();
 
-			Connection connection = intervalTransferer
-					.sendIntervals(intervalsToTransfer);
+			Connection connection = intervalTransferer.sendIntervals(
+					intervalsToTransfer, WatchDogUtils.getWorkspaceName());
```

Comment:
We need projectId, which is stored in preferences and can be fetched with `getOrCreateProjectSetting(projectName)`